src/views/Verbose.vue

```diff
@@ -137,14 +145,18 @@
           this.$store.dispatch('getVerboseData', sessionId);
         }
       }
-    }
+    },
   }
 </script>
 
 <style lang="scss">
   main.verbose {
     display: flex;
     overflow-y: hidden;
+		
+		.demo-button {
+			font-size: 1.5em;
+		}
```

Comment:
Not sure if I shoudl apply these rules to `common.scss` or something like that, suggestions?

Reply:
I think once something is used in multiple components, it should be added to `common.scss`, but it's not hugely important.